W-16451096 Rotating client secrets (#368)
mulesoft/docs-access-management: modules/ROOT/pages/connected-apps-org-admin.adoc

= Connected Apps for Organization Administrators

As an organization administrator at the root organization or business group level, you are the only user who can view and manage connected apps in Access Management. You can use authorization policies to dictate which apps are authorized to access user data, and you can add apps to an allowlist.

== Owned Apps

To manage your connected apps, navigate to *Access Management* and select the *Connected Apps* tab. You can add apps in the *Owned Apps* section and manage app access in the *Authorizations* section.
For more information about the *Owned Apps* section, see the documentation for xref:connected-apps-developers.adoc[app developers].

To maintain security, you can view and manage connected apps only if you have the Organization Administrator permission in the root organization or in the business group where the connected app resides.

> When an organization administrator leaves the organization or is deleted or disabled, the ownership of the connected apps owned by that user is automatically transferred to the organization owner. When ownership is transferred, it is best practice to manually rotate the client secrets of all the connected apps in the organization for security reasons. For more information about how to rotate client secrets, see xref:connected-apps-developers#modify-the-client-secret-for-a-connected-app[Modify the Client Secret for a Connected App].

Reviewer: ```suggestion
When an organization administrator leaves, is deleted, or is disabled, ownership of their connected apps automatically transfers to the organization owner. To maintain security, manually rotate the client secrets of all connected apps in the organization after the transfer. For more information about how to rotate client secrets, see xref:connected-apps-developers.adoc#modify-the-client-secret-for-a-connected-app[Modify the Client Secret for a Connected App].
```

Reviewer: reworded to remove "best practice"
